app/src/main/java/com/infomaniak/mail/ui/main/folder/TwoPaneFragment.kt

```diff
@@ -0,0 +1,137 @@
+/*
+ * Infomaniak Mail - Android
+ * Copyright (C) 2023 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.mail.ui.main.folder
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import androidx.slidingpanelayout.widget.SlidingPaneLayout
+import com.infomaniak.lib.core.utils.getBackNavigationResult
+import com.infomaniak.lib.core.utils.safeNavigate
+import com.infomaniak.mail.R
+import com.infomaniak.mail.ui.MainViewModel
+import com.infomaniak.mail.ui.main.search.SearchFragment
+import com.infomaniak.mail.ui.main.thread.DetailedContactBottomSheetDialogArgs
+import com.infomaniak.mail.ui.main.thread.ThreadFragment
+import com.infomaniak.mail.ui.main.thread.actions.*
+import com.infomaniak.mail.utils.safeNavigateToNewMessageActivity
+
+open class TwoPaneFragment : Fragment() {
```

Comment:
"two-pane" here is an compound adjective created with a number and a noun, so it should use the singular form of the noun.

And it's someone named _"Bodin"_ that said it, so it's definitively true :
https://yolainebodin.com/the-language-nook/english/compound-adjective-with-number-and-noun

=> _"Careful! Remember to keep the noun in the singular form even if the number makes you feel there should be an s for the plural because, as you know, it is now an adjective and there is no s at the end of adjectives in English!"_